src/main/java/uk/gov/hmcts/cmc/claimstore/events/DocumentUploadHandler.java

```diff
@@ -62,9 +63,7 @@ public void uploadDocument(RepresentedClaimIssuedEvent event) {
     public void uploadDocument(DefendantResponseEvent event) {
         Claim claim = event.getClaim();
         requireNonNull(claim, CLAIM_MUST_NOT_BE_NULL);
-        if (!claim.getResponse().isPresent()) {
-            throw new IllegalArgumentException("Response must be present");
-        }
+        claim.getResponse().orElseThrow(() -> new IllegalArgumentException("Response must be present"));
```

Comment:
IllegalstateException or NotFoundException. Pls talk to @srineelesh to be inline with what he agreed.

Reply:
@karthikkannankainos  : I am fixing this in my PR as this is an oversight on my part.

Reply:
Will be addressed in a PR from Srineelesh. So will resolve there.